.ci/configure_podman_for_kata.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
+#
+# Copyright (c) 2020 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+cidir=$(dirname "$0")
+source  "${cidir}/../lib/common.bash"
+
+default_kata_config="/usr/share/defaults/kata-containers/configuration.toml"
+kata_config="/etc/kata-containers/configuration.toml"
+TEST_INITRD="${TEST_INITRD:-no}"
+
+setup_prerequisites() {
+	# Verify host kernel version
+	host_kernel_version=$(uname -r|cut -d. -f1-2)
+	kernel_version="4.14"
+	result=$(echo "${host_kernel_version} >= ${kernel_version}"|bc)
+	[ "${result}" -ne 1 ] && die "Host kernel version is ${host_kernel_version} which is too old"
+	# Disable selinux
+	if  [ "$(getenforce)" != "Disabled" ]; then
+		sudo setenforce 0
+		sudo sed -i 's/^SELINUX=enforcing/SELINUX=disabled/g' /etc/sysconfig/selinux
+	fi
+	# Add user to KVM
+	getent group kvm &>/dev/null || sudo groupadd --system kvm
+	sudo usermod -a -G kvm $USER
+	newgrp kvm << END
+	echo "This is running as group $(id -gn)"
+END
+	sudo chown root:kvm /dev/kvm
+	sudo chmod g+rw /dev/kvm
+}
+
+setup_kata_configuration_files() {
+	sudo mkdir -p $(dirname "${kata_config}")
+	[ ! -e "${kata_config}" ] && sudo install -D "${default_kata_config}" $(dirname "${kata_config}")
+
+	sudo chown root:kvm "${kata_config}"
+	sudo chmod g+r "${kata_config}"
+}
+
+disable_vhost_net() {
+	sudo sed -i -e 's/^#disable_vhost_net = true/disable_vhost_net = true/' "${kata_config}"
+}
+
+modify_kata_image_permissions() {
+	TRUSTED_GROUP=kvm
```

Comment:
I think this variable could be global and use it in L31-37

Reply:
@chavafg changes applied